docs/02/a.md

```diff
@@ -24,7 +24,7 @@ The conventional steps of defining a modular typeclass in Scala used to look lik
 
 Frankly, these steps are mostly copy-paste boilerplate except for the first one.
 Enter Michael Pilquist ([@mpilquist][@mpilquist])'s [simulacrum][simulacrum].
-simulacrum magically generates most of steps 2-4 just by putting `@typeclass` annotation.
+Simulacrum magically generates most of steps 2-4 just by putting `@typeclass` annotation.
```

Comment:
For proper nouns like project names shouldn't we stick to one casing? It should be simulacrum (all lowercase) everywhere or Simulacrum (capitalized) everywhere.